src/serviceWorker.ts

```diff
@@ -2,15 +2,18 @@ import { Color } from './utils';
 
 chrome.runtime.onInstalled.addListener(() => {
   chrome.action.disable();
-  const defaultsColors: Color[] = [
-    { value: '#121212', name: '--primary-bg-color' },
-    { value: '#2a2a2a', name: '--secondary-bg-color' },
-    { value: '#4e4949', name: '--light-bg-color' },
-    { value: '#736868', name: '--very-light-bg-color' },
-    { value: '#29303b', name: '--udemy-darkblue-bg-color' },
-    { value: '#dcc2c2', name: '--primary-font-color' },
-    { value: '#ffffff', name: '--secondary-font-color' },
-    { value: '#ffff00', name: '--success-font-color' },
+  const defaultsColors = [
+    { color: '#121212', name: '--primary-bg-color' },
+    { color: '#1c1d1f', name: '--secondary-bg-color' },
```

Comment:
You made a mistake during resolving conflict . The `color` property does not exist now, you should write `value`